README.md

```diff
@@ -110,6 +110,19 @@ Additionally you can configure many different types of custom Jira field, which
 > It may even break earlier when the requests takes longer then Google's maximum execution timeout.
 > Depending on Jira response time i had successfully listed 1000 issues but sometimes only about >850.
 
+
+### Update Jira Issues from Spreadsheet
```

Comment:
Pls add a _(Beta)_ note to it. As it wont be available for users of regular add-on.

Reply:
Latest when we release this for public, i would like to have you prepare 1 or 2 screenshots or even an short video for the Marketplace description.

Reply:
https://github.com/ljay79/jira-tools/issues/137